boards/arm/sama5/giant-board/src/sam_ethernet.c

```diff
@@ -0,0 +1,328 @@
+/****************************************************************************
```

Comment:
This should be removed.  The EMAC peripherals are not brought out on the Giant Board.  So this file is inappropriate and misleading.

Reply:
@patacongo Same comment as above re: Ethernet over USB. I will look into exactly how that's implemented. Will remove if I don't need it for Ethernet over USB.

Reply:
If you need anything releated to the non-function EMAC/GMAC peripheral then something is horribly screwed up in my implementation.  USB CDC/ECM absolutely does not need it and the perpherial is non-functional since the pins are not exposed.

Reply:
You understand that the Ethernet MAC that is used resided the the USB device that is connected to SAMA5 via USB host.  That is the MAC used by the network, not the SAMA5 MAC which you could not use if you wanted to.

Reply:
@patacongo Ok, will leave HAVE_ETHERNET and remove the other EMAC/GMAC references. Thank you for clarifying.

Reply:
It is only used in sam_ethernet.c which should be removed anyway

Reply:
@patacongo Ok, will search for references, and remove if I can.